docs/controls/file-pickers/node/index.md

```diff
@@ -0,0 +1,75 @@
+---
+author: keco
+ms.author: keco
+ms.date: 10/02/2018
+ms.topic: overview
+---
+# Microsoft File Browser SDK for Node.js (Preview)
+
+The [Microsoft File Browser SDK](https://www.npmjs.com/package/@microsoft/file-browser) provides a reusable React component for building file experiences with the [Microsoft Graph](https://developer.microsoft.com/en-us/graph). By using the SDK, any React application can render and perform actions on files by providing a valid `access_token`. In this guide, we will show you how to get your React app quickly rendering the File Browser component for Microsoft Graph.
+
```

Comment:
Will we want to eventually link to the sample GitHub project here?

Reply:
For sure, we might as well flip the switch prior to publishing this.

Reply:
How should we link it?

>Additional examples can be found in <repo>

? Let me know your thoughts.

Reply:
Maybe

>You can also follow along with our [file browser sample project](link)

similar to what we have in the last sentence of the second paragraph [here](https://github.com/OneDrive/onedrive-api-docs/blob/ba68723adf49570d7d4cc1261f57cff82f9610b1/docs/controls/file-pickers/android/index.md#onedrive-picker-for-android).